pkg/grpcserver/device.go

```diff
@@ -176,6 +178,19 @@ func (s *Server) GetDevice(ctx context.Context, request *dmiapi.GetDeviceRequest
 		}
 		res.Device.Status.Twins = twins
 		res.Device.Status.State = common.DEVSTOK
+	case common.ProtocolCustomized:
+		deviceValue := reflect.ValueOf(device)
+		//d := device.(*customizedDriver.CustomizedDev)
```

Comment:
Is it possible to remove it?

Reply:
To make this a universal framework. I plan to delete all switch/case structures and not import the corresponding device's packages, and subsequently obtain the corresponding fields through reflection. I will modify it later and hope you can continue to provide valuable feedback. Thanks! :)

Reply:
sorry, but I mean `//d := device.(*customizedDriver.CustomizedDev)` this unused code, Is it possible to remove it?

Reply:
Done